### PR TITLE
Kh360 react 527 progress screen overlapping

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Evaluation Progress - overlapping issue on Evaluee details
[#527](https://github.com/nerubia/kh360-react/issues/527)

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -36,6 +36,9 @@ module.exports = {
           500: "#b7e0ff",
         },
       },
+      width: {
+        "90": "90%"
```
Comment (emmaacha) [inline tailwind.config.js:40]: I don't think this is needed, when using custom size for width, you can do `w-[90%]`

File: src/features/admin/evaluation-templates/create/evaluation-template-form.tsx
Comment (emmaacha) [inline src/features/admin/evaluation-templates/create/evaluation-template-form.tsx:1]: What are these changes for?

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/features/admin/evaluation-templates/create/evaluation-template-form.tsx
Comment (emmaacha) [reply src/features/admin/evaluation-templates/create/evaluation-template-form.tsx:1]: I think some changes from the other commits/PRs are being added here, kindly check for these changes and exclude in this PR, thanks!

File: tailwind.config.js (hunk 1 above)
Comment (chadiegilnerubia) [reply tailwind.config.js]: done :3

File: src/features/admin/evaluation-templates/create/evaluation-template-form.tsx
Comment (chadiegilnerubia) [reply src/features/admin/evaluation-templates/create/evaluation-template-form.tsx:1]: done :3

**Review by emmaacha — APPROVED** [review]

**Review by gborgonia-ir — APPROVED** [review]

**Pull request merged**